src/main/java/net/commments/object/CommentListActions.java

```diff
@@ -0,0 +1,11 @@
+package net.commments.object;
+
+public interface CommentListActions {
```

Comment:
Why did you create this interface? It's not used in the test. Please remove.

Reply:
@lvrub please pay attention to this comment.

Reply:
done